nearby-app/README.md

```diff
@@ -39,42 +65,24 @@ outputAttributes.add("*");
 // Execute the search[]
 <ListenableFuture> results = mLocatorTask.geocodeAsync(searchText, parameters);
 ```
+A distance and bearing from the device's location is calculated for each returned place.
 
-## Device Location
-This app uses a [mapless app pattern](https://developers.arcgis.com/android/guide/determine-your-app-map-pattern.htm#ESRI_SECTION1_58C46384E3484890A47629F8F12E6EF5) with a list of places and option to see the list on a map. Since the app starts with a list, rather than a map, the device location is obtained using Google’s Location Services API. In the future, the Runtime SDK can be used to obtain the device location outside of the MapView.
-
-```java
-// Google's location services are configured in the
-// PlacesFragment onCreate method.
-if (mGoogleApiClient == null) {
-    // Create an instance of GoogleAPIClient.
-    mGoogleApiClient = new GoogleApiClient.Builder(getActivity())
-      .addConnectionCallbacks(this)
-      .addOnConnectionFailedListener(this)
-      .addApi(LocationServices.API)
-      .build();
-}
-// Once Google's location service is connected,
-// you can use the device location to start the 
-// geocoding search.
-@Override public void onConnected(@Nullable Bundle bundle) {
-    mLastLocation = LocationServices.FusedLocationApi.getLastLocation(mGoogleApiClient);
-    startSearch(mLastLocation);
-}
-```
-
-## Calculating Bearing and Distance
+### Calculating Bearing and Distance
 To determine distance and bearing, the [geometry engine](https://developers.arcgis.com/android/latest/api-reference/reference/com/esri/arcgisruntime/geometry/GeometryEngine.html) is used to calculate the [geodesic distance](https://geonet.esri.com/groups/coordinate-reference-systems/blog/2014/09/01/geodetic-distances-how-long-is-that-line-again) between the device location and each nearby point of interest. Measuring distance, determining spatial relationships, and altering geometries can be done locally on the mobile client.
 
-The current device location requires a spatial reference when calculating distance and bearing.
-
+When using the current device location in geospatial calculations, a spatial reference for the device location must be defined since the location returned by Google's location service doesn't specify a spatial reference.
```

Comment:
Duplicate usage of *device location*, remote 2nd instance
> When using the current device location in geospatial calculations, a spatial reference must be defined since the location returned by Google's location service doesn't specify a spatial reference.